test/images/scripts/virt-builder-fedora

```diff
@@ -32,6 +32,8 @@ if [ "$#" -gt 3 ]; then
     subscription_args="--mkdir /root/.rhel \
       --upload $subscription_path/login:/root/.rhel/login \
       --upload $subscription_path/pass:/root/.rhel/pass"
+else
+    subscription_args="--touch /root/.skip_repos"
```

Comment:
This is here so that CentOS skips the RHEL specific repository setup, right?  This wsn't immediately clear to me.  What about making this clearer by turning this around into a positive flag: Create /root/.subscribe-to-repos when we want rhel-7.setup to subscribe and setup the repos.  Or just use the existence of /root/.rhel as the flag.